gateway/service/src/main/java/org/eclipse/ditto/gateway/service/endpoints/routes/thingsearch/ThingSearchRoute.java

```diff
@@ -96,13 +114,43 @@ private Route searchThings(final RequestContext ctx, final DittoHeaders dittoHea
         //           &fields=<fieldsString>
         //           &namespaces=<namespacesString>
         //           &nextPageKey=<nextPageKey>
-        return get(() -> thingSearchParameterOptional(params -> handlePerRequest(ctx,
-                QueryThings.of(calculateFilter(params.get(ThingSearchParameter.FILTER)),
-                        calculateOptions(params.get(ThingSearchParameter.OPTION)),
-                        AbstractRoute.calculateSelectedFields(params.get(ThingSearchParameter.FIELDS))
-                                .orElse(null),
-                        calculateNamespaces(params.get(ThingSearchParameter.NAMESPACES)),
-                        dittoHeaders))));
+        // POST things?options=<optionsString>PUT
```

Comment:
`POST things?options=<optionsString>PUT` ? 
What is that supposed to mean?

As mentioned above, I would not "mix" query params with form params of the POST - and provide all query params as form values instead.

Reply:
Typo